src/lsi/lsi.py

```diff
@@ -612,6 +621,29 @@ def main(progname=sys.argv[0]):
         attribs = HostEntry.list_attributes()
         print 'The following attributes are available: {}'\
                     .format(', '.join(attribs))
+    elif args.ansible is not None:
+        latest = True
+        inventory_list = {}
```

Comment:
Why is this being used? It looks like there's only one `query_group` being made anyway

Also, it's not a list